### ISSUE TITLE
Broken colab tutorial link?

### ISSUE THREAD

**Issue by igarizio** [signalling-games-org/evoke#72]
Hi @stephenfmann,

I think the [colab tutorial link](https://colab.research.google.com/drive/1AwUCP05lpITAP7_EZD7loGv3unhnwvhM#forceEdit=true&sandboxMode=true) in the README, docs, etc is broken (or at least I can't access it).
![drive](https://github.com/user-attachments/assets/2a84e82f-229d-4564-872b-443bc210a584)

Thanks!

**Comment by stephenfmann** [comment]
Yep it's gone missing -- I will look into this now. Thanks.

**Comment by stephenfmann** [comment]
I've restored the notebook but it will now have a different URL. You should be able to access it [here](https://colab.research.google.com/drive/1zFRb20KoQi0Tdg5UR1_zReHCZicnQLZd#forceEdit=true&sandboxMode=true). I will update the URL in the various places where it's listed, and then close the issue. 

Thanks for this! Let me know if there are any more problems.

**Comment by stephenfmann** [comment]
Links updated in README and documentation. The link on the PyPI package page will be updated next time there's a version release, which is planned for after the JOSS review process.

**Comment by igarizio** [comment]
Hi, sorry to comment again, but now there is a permission issue ("You need access... Request access, or switch to an account with access.").

My apologies for not suggesting this earlier, but I think it would be a good idea to have your notebook saved in the repository (as an `.ipynb` file). This will not only resolve any access issues but will let us have an archived copy of it.
Once you have the file, you can update all the colab links like this (and everything should work the same):
- Example notebook: `https://github.com/jupyter/notebook/blob/main/docs/source/examples/Notebook/Running%20Code.ipynb`
- Colab link: `https://colab.research.google.com/github/jupyter/notebook/blob/main/docs/source/examples/Notebook/Running%20Code.ipynb`

In other words, you put `https://colab.research.google.com/` + github link removing `https://` and `.com`.

Please let me know what you think. Thanks again!

**Comment by stephenfmann** [comment]
Thanks, the access issue should be fixed now. Could you check again?

I'll have a look at embedding the notebook in the repository. The last time I tried there were some frustrating issues, e.g. git wanted to sync all of the outputs of the cells after I had finished browsing the notebook. But maybe there are ways around that kind of behaviour.

**Comment by stephenfmann** [comment]
Thanks, I've opened a new issue to investigate moving the notebook into the repository. I'll close this issue now (but feel free to reopen it if you have any more issues!)
